templates/email/messages/berth_offer_new_en.html

```diff
@@ -192,7 +192,7 @@ <h3>Price of the boat berth</h3>
             <p>1. Notify us via this link:</p>
 
             <a style="display: block; padding: 16px; color: #0000BF; background-color: transparent; border: solid 2px #0000BF; text-align: center; text-decoration: none;"
-               href="{{ cancel_url }}">No thank you, I do not wish to reserve
```

Comment:
For these I would remove the whole link (or comment them out, but I'm not sure if all emails clients can handle commented html). If invoices are sent out with broken cancellation links customers will be confused. 

Alternatively replace with text that says something along the lines of "If you do not want to reserve this berth, please contact <vene office email>"

Reply:
Ok ok, I was thinking that I wouldn't touch the template much since it's only a temporary fix until we get the cancel page, but I guess we can comment out the whole section. 👍

Reply:
agree

Reply:
@ahie fixed 👍

Reply:
👍